tests/physics/step-18-rotation_matrix.cc

```diff
@@ -400,9 +400,10 @@ namespace Step18
           cell_matrix = 0;
           cell_rhs    = 0;
           fe_values.reinit(cell);
-          for (unsigned int i = 0; i < dofs_per_cell; ++i)
-            for (unsigned int j = 0; j < dofs_per_cell; ++j)
-              for (unsigned int q_point = 0; q_point < n_q_points; ++q_point)
+          for (const unsigned int i : fe_values.local_dof_indices())
+            for (const unsigned int j : fe_values.local_dof_indices())
+              for (const unsigned int q_point :
+                   fe_values.quadrature_point_indices())
```

Comment:
Ah, such a shame that it breaks this line...

Reply:
You're going to make your life shorter by agonising over what clang-tidy does to our code :-D

Reply:
Now that is a valid point. I think I still quite haven't gotten over the carnage clang-format has wreaked onto our code base...